### PR TITLE
2 integrate embriotech progress

### PR DESCRIPTION
**Pull request by filo87**
Related to #2

### CODE REVIEW

**Comment by filo87** [comment]
@offerijns Ready for review

File: README.md [hunk 1]
```diff
@@ -1,4 +1,14 @@
-# Centrifuge Pools Subquery
+# Centrifuge Subquery
+
+[![SubQuery](https://img.shields.io/static/v1?label=built+with&message=SubQuery&color=FF4581)](https://subquery.network/)
+[![embrio.tech](https://img.shields.io/static/v1?label=by&message=EMBRIO.tech&color=24ae5f)](https://embrio.tech)
+[![Centrifuge](https://img.shields.io/static/v1?label=for&message=Centrifuge&color=2762ff)](https://centrifuge.io/)
```
Comment (offerijns) [inline README.md:5]: Could you remove these lines? The subquery will be contributed to by EMBRIO, but of course not entirely built by EMBRIO either. This does imply the latter.

```suggestion
```

File: README.md [hunk 2]
```diff
@@ -1,4 +1,14 @@
...
+
+<!-- [![Docker](https://img.shields.io/static/v1?label=shipped+with&message=Docker&color=287cf9)](https://www.docker.com/) -->
+
+A GraphQL API to query the Centrifuge Parachain on Polkadot.
+
+🚧 We are breeding the next EMBRIO. More soon... 🚀
```
Comment (offerijns) [inline README.md:11]:
```suggestion
```

File: schema.graphql [hunk 3]
```diff
@@ -6,8 +11,9 @@ type Pool @entity {
   # pools, so we can do Pool.getByType("POOL").
   type: String! @index
 
-  createdAt: Date!
-  
+  createdAtTimestamp: Date!
+  createdAtHeight: Int!
```
Comment (offerijns) [inline schema.graphql:15]: Height is a bit unclear to me, I'm assuming that's the block number?

```suggestion
  createdAt: Date!
  createdAtBlockNumber: Int!
```

File: schema.graphql [hunk 4]
```diff
@@ -19,79 +25,96 @@ type Pool @entity {
...
-  trancheStates: [TrancheState] @derivedFrom(field: "poolState")
-}
-
-type HourlyPoolState @entity {
-  id: ID!
-  
   timestamp: Date!
-  poolState: PoolState!
-}
+  blockHeight: Int!
```
Comment (offerijns) [inline schema.graphql:103]:
```suggestion
  blockNumber: Int!
```

File: src/helpers/stateSnapshot.ts [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+}
+
+interface GenericSnapshot {
+  id: string
+  timestamp: Date
+  blockHeight: number
+  save(): Promise<void>
+}
+
+export async function stateSnapshotter<
```
Comment (offerijns) [inline src/helpers/stateSnapshot.ts:23]: This works nicely!

**Review by filo87 — COMMENTED** [review]
@offerijns tryed to consistently rename all blochHeight instances to blockNumber...

**Review by filo87 — COMMENTED** [review]
@offerijns Ok I just resolved the comments and tested locally...and the indexer was running..Do you want do run it locally too and then we merge?

**Comment by filo87** [comment]
@offerijns  This one should fix the issues with the charts. can you try it?

In the data model you will also find some fields names now ending with a _ underscore. Such fields are automatically reset to 0 by the `stateSnapshotter()` at the end of a period when taking a snapshot. I thinks this simplifies the handling of aggregated values.

 Hope it does not conflict with your current implementations.

**Review by offerijns — APPROVED** [review]

**Pull request merged**